irohad/main/application.cpp

```diff
@@ -161,8 +152,11 @@ Irohad::RunResult Irohad::init() {
   };
 
   // clang-format off
-  return initWsvRestorer() // Recover WSV from the existing ledger
-                           // to be sure it is consistent
+  return initSettings()
```

Comment:
`initStorage()` in constructor uses `block_validators_config_`, therefore settings and validators configs should be initialized before storage. Please move these calls to constructor.

Reply:
This config uses for AlwaysValidValidator
```C++
AlwaysValidValidator() = default;
AlwaysValidValidator(
    std::shared_ptr<shared_model::validation::ValidatorsConfig>){};
shared_model::validation::Answer validate(const T &m) const override {
    return {};
}
```
But here can use Default constructor instead of the constructor with config